test

```diff
@@ -118,10 +118,13 @@ title "environment / protect FOO even with -env"
 }
 
 title "user / default"
-{
-	result=$(FRESH_INTERNAL_RUNCMD='/usr/bin/id -un' ./bin/fresh-node10)
-	assert_lastline "$result" "nobody"
```

Comment:
Preserving that we logon as `nobody` on macOS still seems useful to assert I think?

Reply:
yeah, that's fair. I can bring that back.